src/opcodes/opcodes.cairo

```diff
@@ -388,4 +389,8 @@ pub mod Opcode {
         let a = engine.astack.pop_byte_array();
         engine.dstack.push_byte_array(a);
     }
+
+    fn opcode_reserved(ref engine: Engine) {
+        format!("attempt to execute reserved opcode");
```

Comment:
I believe this should be using `panic!` to throw an error for now, this is just formatting a string.